### ISSUE TITLE
Passing Environment Variables with qsub

### ISSUE THREAD

**Issue by gcamilo** [JuliaParallel/ClusterManagers.jl#27]
My cluster has an old compiler, so I compiled GCC in my user directory, so I had to add to LD_LIBRARY_PATH my lib directory. Now launching workers does not work:

```
/home/---/julia/usr/bin/julia: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /home/---/julia/usr/bin/../lib/libjulia.so)
```

As the environment variables are not passed to the launched workers. Adding a ```-V``` option to the qsub command fixes this. Should we add this as a default?

**Comment by gcamilo** [comment]
@amitmurthy

**Comment by amitmurthy** [comment]
Wouldn't it be better to use `-v` and specify only the env variables you need? `params` contains all the keyword args. You could pass a `qsubenv = Dict("ENV1"=>"VAL1",....)` and process them as `-v` options.

**Comment by gcamilo** [comment]
So adding another argument to ```addprocs``` where you can optionally choose which environment variables to pass?

**Comment by amitmurthy** [comment]
Yes. All keyword arguments to `addprocs` are available in `params` in 
`launch(manager::Union{PBSManager, SGEManager}, params::Dict, ..........)`

**Comment by gcamilo** [comment]
So I thought of modify ```addprocs_sge``` definition:

```
addprocs_sge(np::Integer, queue::AbstractString="",enviromentVars::AbstractString="") = addprocs(SGEManager(np, queue),(:enviromentVars,enviromentVars))
```

And then modifying launch to have:

```
function launch(manager::Union{PBSManager, SGEManager}, params::Dict, instances_arr::Array, c::Condition)
   ...
  enviromentVars = params[:enviromentVars]
        if enviromentVars == ""
            eVars = ""
        else
            eVars = ["-v" enviromentVars]
        end
```
And adding this to the open command. What do you think?

**Comment by amitmurthy** [comment]
Yes, this should work.

**Comment by gcamilo** [comment]
Ok, I'll amend my fixQueue pull request to make it take care of this.

**Comment by gcamilo** [comment]
fixed with #33
